File: app/src/main/java/com/durbar/bangabandhuplay/data/model/category/sub/all/RootCategory.java
package com.durbar.bangabandhuplay.data.model.category.sub.all;

import androidx.annotation.Nullable;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RootCategory {
    @Nullable
    @SerializedName("id")
    @Expose
    private Integer id;
    @Nullable
    @SerializedName("title")
    @Expose
    private String title;
    @Nullable
    @SerializedName("slug")
    @Expose
    private String slug;
    @Nullable
    @SerializedName("image")
    @Expose
    private String image;
    @Nullable
    @SerializedName("order")
    @Expose
    private Integer order;
    @Nullable
    @SerializedName("seo_title")
    @Expose
    private String seoTitle;
    @Nullable
    @SerializedName("seo_description")
    @Expose
    private String seoDescription;
    @Nullable
    @SerializedName("status")
    @Expose
    private String status;
    @Nullable
    @SerializedName("created_at")
    @Expose
    private String createdAt;
    @Nullable
    @SerializedName("updated_at")
    @Expose
    private String updatedAt;


    public RootCategory(Integer id, String title, String slug, String image, Integer order, String seoTitle, String seoDescription, String status, String createdAt, String updatedAt) {
        this.id = id;
        this.title = title;
        this.slug = slug;
        this.image = image;
        this.order = order;
        this.seoTitle = seoTitle;
        this.seoDescription = seoDescription;
        this.status = status;
        this.createdAt = createdAt;
        this.updatedAt = updatedAt;
    }
}

File: app/src/main/java/live/durbar/bangabandhuapp/data/model/category/sub/all/RootCategory.kt
package live.durbar.bangabandhuapp.data.model.category.sub.all

import com.google.gson.annotations.SerializedName

data class RootCategory(
    @field:SerializedName("id")
    val id: Int? = 0,

    @field:SerializedName("title")
    val title: String? = null,

    @field:SerializedName("slug")
    val slug: String? = null,

    @field:SerializedName("image")
    val image: String? = null,

    @field:SerializedName("order")
    val order: Int? = 0,

    @field:SerializedName("seo_title")
    val seoTitle: String? = null,

    @field:SerializedName("seo_description")
    val seoDescription: String? = null,

    @field:SerializedName("status")
    val status: String? = null,

    @field:SerializedName("created_at")
    val createdAt: String? = null,

    @field:SerializedName("updated_at")
    val updatedAt: String? = null
)
